LogMaker/FunctionTracer/CallTracer.h

```diff
@@ -0,0 +1,13 @@
+#include "pin.H"
+using std::string;
+
+#define LIBC_BASE 0x700000000000
```

Comment:
Time to get rid of this hack, lets also try to unify the modules so code is not repeated. what say @pranjalsingh008?

Reply:
Okay, We did implement a method of finding the correct addresses of where each of the libraries are loaded. But then we face another issue/dilemma. Its whether we really need to store the base addresses of each and every library mapped ?. If yes then this and the next issue will also be resolved.

Reply:
I remember seeing a solution in one of the CTF challenges where they gave a pin tool and made some sort of trace. @akulpillai do you remember this?

Reply:
nope, anyway I talked to @Freakston regarding making this into a single binary. He has looked into it, they will work on this once all modules near completion.

Reply:
SECCON 2019 - follow-me was the challenge